### PR TITLE
bugfixes prometheus

### PR DESCRIPTION
**Pull request by lennartvanlaake**

### CODE REVIEW

**Review by aapeliv — DISMISSED** [review]
LGTM

File: app/prometheus/entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 /bin/sed -i 's/{GRAFANA_CLOUD_TOKEN}/'$GRAFANA_CLOUD_TOKEN'/' /etc/prometheus/prometheus.yml
```
Comment (aapeliv) [inline app/prometheus/entrypoint.sh:2]: Should this also needs curly braces?

**Review by aapeliv — APPROVED** [review]

File: app/prometheus/entrypoint.sh (hunk 1 above)
Comment (lennartvanlaake) [reply app/prometheus/entrypoint.sh:2]: it should not have curly braces, the PROMETHEUS_ENVIRONMENT one is wrong

Comment (lennartvanlaake) [reply app/prometheus/entrypoint.sh:2]: (fixed in last commit)

**Pull request merged**